src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,18 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
```

Comment:
Since this component is otherwise totally generic, could we make this prop something more generic like `optionGroupLink` or something?

Reply:
Eh tbh maybe not necessary since this is a project selector component 😄 your call!

Reply:
I agree with @sophstad  I think trying to make this a little more generic would be nice. It would also alleviate some of the conditional logic that happens below.

Reply:
I'm assuming you are referring to isProjectSettingsPage and not repoIdentifier?